src/views/Rules/Rule/index.jsx

```diff
@@ -425,10 +549,10 @@ export default function Rule({ isNewRule, ...props }) {
               <ContentSaveIcon />
             </Fab>
           </Tooltip>
-          {scheduleChange.data && scheduleChange.data.data.count > 0 && (
+          {hasScheduledChange && (
             <SpeedDial ariaLabel="Secondary Actions">
               <SpeedDialAction
-                disabled={scheduleChange.loading}
+                disabled={scheduledChangeAction.loading}
                 icon={<DeleteIcon />}
                 tooltipOpen
                 tooltipTitle="Cancel Pending Change"
```

Comment:
STR:

1. Navigate to http://localhost:9000/rules/create
2. Click the save icon button

Expected: An error complaining about one of the required fields not being there
Actual: The error panel shows `update_type: '' is not one of ['major', 'minor', None]`. 2 issues:
1. Update Type should have the `required` prop
2. Update Type should be a menu with inputs "-", 'Minor', 'Major'.

I understand this issue is out of scope from the mutation actions so feel free to do this in a follow-up or we could simply create an issue for it to make sure it is tracked :)